packages/teleport/src/services/auth/auth.test.js

```diff
@@ -33,6 +33,9 @@ describe('services/auth', () => {
   });
 
   // sample data
+  const dummyU2fRegChallenge = Promise.resolve({
+    u2f: { appId: 'xxx' },
+  });
```

Comment:
Bit of a nitpick but I think it would be more readable to have this be just the actual object (without the `Promise.resolve()`) and then when you need it, use `.mockResolvedValue(dummyU2fRegChallenge)` instead of `mockImplementation(() => dummyU2fRegChallenge)`